Assets/Puppet/PuppetReleaseHandler.cs

```diff
@@ -7,20 +7,22 @@
 // Very simple release handler.
 public class PuppetReleaseHandler : MonoBehaviour
 {
-    private static ReleaseDetails _releaseDetails = null;
+    private static ReleaseDetails _releaseDetails;
     private static readonly object _releaseLock = new object();
+    public static bool IsDialogCustom;
     public PuppetUpdateDialog Dialog;
 
     void Awake()
     {
         Distribute.ReleaseAvailable = OnReleaseAvailable;
+        IsDialogCustom = PlayerPrefs.GetInt(PuppetAppCenter.FlagCustomDialog, 0) == 1;
```

Comment:
I think it can be simplified to
```suggestion
        IsDialogCustom = PlayerPrefs.GetInt(PuppetAppCenter.FlagCustomDialog, 0);
```

Reply:
...And the build is broken 😄